R/derive_vars_dtm_to_dt.R

```diff
@@ -35,11 +35,28 @@
 #'   )
```

Comment:
Sorry i cannot comment on the grey code so, adding them here:
- (line1) The title should be Title Case.
- (line 3) Should we update as :This function creates date(s) as output from datetime variable(s)
- (line 11-13) : Should we update as : A data frame containing the input dataset with the corresponding date (`--DT`) variable(s) of all datetime variables (`--DTM`) specified in `source_vars`.
- line 24-30: i would suggest to remove `ASTTMF`